### DISCUSSION TITLE
Oh, hi.

### DISCUSSION THREAD

**Discussion by mszoek** [ravynsoft/ravynos#91] [General]
Welcome to Airyx!

I just wanted to point out that we have this cool little Discussions tab where you can ask questions without opening issues :)

**Comment by wesley974** [comment]
Why the choice FreeBSD and not OpenBSD?

**Reply by mszoek** [reply]
Well... FreeBSD is the basis of macOS, so that's one reason. It also provides compatibility to Linux apps which allows us to run some things that aren't available for FreeBSD. It's going to be a long time before Airyx can run a majority of Cocoa apps or macOS binaries so broad compatibility until then is important. Then there's ZFS which provides similar features to APFS - it's not available in OpenBSD AFAIK. The ability for unprivileged users to mount volumes was also removed from OpenBSD and I need that. 

While I appreciate OpenBSD's philosophy and it is no doubt secure AF, it felt like the wrong choice for what I'm trying to do.

**Reply by wesley974** [reply]
That makes sense, thanks!

**Comment by MkStevo** [comment]
I'd like to offer congratulations to you. I wish you the best of luck and I'll be keeping an eye on the progress. I absolutely loved my iMac but found the restrictions the Apple started to impose on the OS prevented me from upgrading due to no longer being able to run 32bit Applications. As I couldn't get 64bit versions of some vital Applications I became stuck on an out dated OS. Eventually I had to switch back to a Windows machine to maintain productivity. Now I hate Microsoft again, but at least I can get work done. Yuk!

I've started to ramble, so I'll just repeat my thanks, offer you my congratulations again and wish you well.

**Reply by mszoek** [reply]
Thank you! Hopefully one day we can provide a good alternative :)

**Comment by szcsaba101** [comment]
Hi dear Zoe!
I've been using computers for 23 years, but I've never been able to replace windows. But I am interested in other alternative systems. I've tried linux, bsd, mac. I’m not an IT just an average user.
I would be glad if your system matured. Reactos e.g. it has never become mature software and is not good.
best regards
Csaba

**Reply by mszoek** [reply]
Thanks! It's a labor of love for me and I hope someday it will help others :)
